core/storage/suggestion/gae_models.py

```diff
@@ -319,6 +329,42 @@ def get_all_stale_suggestion_ids(cls):
                 cls.last_updated < threshold_time).fetch()
         return [suggestion_model.id for suggestion_model in suggestion_models]
 
+    @classmethod
+    def get_suggestions_waiting_too_long_for_review(cls):
+        """Returns a list of suggestions that have been waiting for a review
+        longer than SUGGESTION_REVIEW_WAIT_TIME_THRESHOLD_IN_DAYS days on the
+        Contributor Dashboard. MAX_NUMBER_OF_SUGGESTIONS_TO_EMAIL_ADMIN
+        suggestions are returned, sorted in descending order by their review
+        wait time.
+
+        Returns:
+            list(GeneralSuggestionModel). A list of suggestions, sorted in
+            descending order by their review wait time.
+
+        Raises:
+            Exception. If there are no suggestion types offered on the
+                Contributor Dashboard.
+        """
+        if not CONTRIBUTOR_DASHBOARD_SUGGESTION_TYPES:
+            raise Exception(
+                'Expected the suggestion types offered on the Contributor '
+                'Dashboard to be nonempty.')
+        threshold_time = (
+            datetime.datetime.utcnow() - datetime.timedelta(
+                days=SUGGESTION_REVIEW_WAIT_TIME_THRESHOLD_IN_DAYS))
+        return (
+            cls.get_all()
+            .filter(
+                cls.status == STATUS_IN_REVIEW)
```

Comment:
I don't think you need to break after '(', here and below. You can put each filter/fetch on its own line (except for the ones that span over a line)?

Reply:
Done